src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -134,13 +144,46 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
     };
   }, [relatedVenueEvents, userEventIds, selectedDayIndex, getEventLocation]);
 
+  const downloadPersonalEventsCalendar = useCallback(() => {
+    const dayStart = addDays(startOfToday(), selectedDayIndex);
+    const allPersonalEvents: PersonalizedVenueEvent[] = relatedVenueEvents
+      .map(
+        prepareForSchedule({
+          day: dayStart,
+          usersEvents: userEventIds,
+          isForCalendarFile: true,
+        })
+      )
+      .filter((event) => event.isSaved);
+
+    downloadCalendar({
+      calendarName: `${PLATFORM_BRAND_NAME}_Personal`,
+      events: allPersonalEvents,
+    });
```

Comment:
```suggestion
    downloadCalendar(createCalendar({
      calendarName: `${PLATFORM_BRAND_NAME}_Personal`,
      events: allPersonalEvents,
    }));
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1423/commits/1346ad92bc3006ba3bbbe1f5695c2806dcc4f4c4